tools/shared/pragma.c

```diff
@@ -658,7 +715,14 @@ do_sw(void)
           break;
         }
         LCASE(ctok);
-        if (strcmp(ctok, "always") == 0) {
+        if (strcmp(ctok, "vectorlength") == 0) {
+          int toSet = 0x0;
+          int Num = -1;
```

Comment:
Keep the naming convention of the project: object type variable names start with capital letter, simple type variable names start with lower case letter, should be `int num` not `in Num`.